docker/1.3.1/py2/Dockerfile.cpu

```diff
@@ -0,0 +1,80 @@
+FROM ubuntu:16.04
+
+LABEL maintainer="Amazon AI"
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    ca-certificates \
+    cmake \
+    curl \
+    git \
+    wget \
+    vim \
+    jq \
+    libsm6 \
+    libxext6 \
+    libxrender-dev \
+    build-essential \
+    zlib1g-dev \
+    libglib2.0-0 \
+    libgl1-mesa-glx \
+    openjdk-8-jdk-headless
+
+# See http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
+
+ENV LD_LIBRARY_PATH /opt/conda/lib/:$LD_LIBRARY_PATH
+
+# Add arguments to achieve the version, python and url
+ARG PYTHON_VERSION=2.7
+
+RUN curl -o ~/miniconda.sh -O  https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh  && \
+     chmod +x ~/miniconda.sh && \
+     ~/miniconda.sh -b -p /opt/conda && \
+     rm ~/miniconda.sh && \
+     /opt/conda/bin/conda update conda && \
+     /opt/conda/bin/conda install -y python=$PYTHON_VERSION \
+     numpy==1.16.4 \
+     scipy==1.2.1 \
+     ipython==5.8.0 \
+     mkl==2019.4 \
+     mkl-include==2019.4 \
+     cython==0.29.12 \
+     typing==3.7.4 && \
```

Comment:
Update all libraries to latest versions

Reply:
Easiest way to do this is to unpin all versions, run the build, if it's successful, pin versions according to the ones that come up in a `pip freeze | grep <package>` within the container.